test/src/integration/integration_test.go

```diff
@@ -165,14 +170,36 @@ func runAnsiblePlaybooks(t *testing.T) {
 	}
 
 	// Install tools
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventory.ini", []string{"./playbooks/playbook-install-tools.yml", "-e", "base_local_directory=../../../../", "-e", "install_awscli=" + installAwscli})
 
 	// Deploy scalardl
-	runAnsiblePlaybook(t, k8sModuleDir, []string{"./playbooks/playbook-deploy-scalardl.yml", "-e", "base_local_directory=../../../conf"})
+	runAnsiblePlaybook(t, k8sModuleDir, "../inventory.ini", []string{"./playbooks/playbook-deploy-scalardl.yml", "-e", "base_local_directory=../../../conf"})
+}
+
+func runAnsiblePlaybooksWithGoss(t *testing.T, targetModules []string, targetHosts string) {
+	cloudProvider := "aws"
+	if strings.Contains(*terraformDir, "azure") {
+		cloudProvider = "azure"
+	}
+
+	err := ioutil.WriteFile("./ssh.cfg", []byte(lookupTargetValue(t, "network", "ssh_config")), 0644)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, m := range targetModules {
+		err = ioutil.WriteFile("./inventories/"+m, []byte(lookupTargetValue(t, m, "inventory_ini")), 0644)
```

Comment:
```suggestion
		err = ioutil.WriteFile("./inventories/" + m, []byte(lookupTargetValue(t, m, "inventory_ini")), 0644)
```

Reply:
This format seems to have been fixed by `go fmt` locally for me. Can you try it?

Reply:
Sorry, it was formatted correctly with `go fmt`. 👍